Dockerfile

```
@@ -495,6 +493,54 @@ RUN \
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-go as sdk-govc
+
+USER root
+RUN \
+  mkdir -p /usr/libexec/tools /usr/share/licenses/govmomi && \
+  chown -R builder:builder /usr/libexec/tools /usr/share/licenses/govmomi
+
+ARG ARCH
+ARG TARGET="${ARCH}-bottlerocket-linux-gnu"
+ARG GOVMOMIVER="0.26.0"
+ARG GOVMOMISHORTCOMMIT="34586b6"
+
+USER builder
+WORKDIR ${GOPATH}/src/github.com/vmware/govmomi
+COPY ./hashes/govmomi /home/builder/hashes
+RUN \
+  sdk-fetch /home/builder/hashes && \
+  tar --strip-components=1 -xf govmomi-${GOVMOMIVER}.tar.gz && \
+  rm govmomi-${GOVMOMIVER}.tar.gz
+
+ARG GOARCH_aarch64="arm64"
+ARG GOARCH_x86_64="amd64"
+ARG GOARCH_ARCH="GOARCH_${ARCH}"
+
+COPY --chown=0:0 --from=sdk-license-scan /usr/libexec/tools/ /usr/libexec/tools/
+COPY --chown=0:0 --from=sdk-license-scan /home/builder/license-scan/clarify.toml .
+RUN \
+  cp -p LICENSE.txt /usr/share/licenses/govmomi && \
+  go mod vendor && \
+  /usr/libexec/tools/bottlerocket-license-scan \
+    --clarify clarify.toml \
+    --spdx-data /usr/libexec/tools/spdx-data \
+    --out-dir /usr/share/licenses/govmomi \
+    go-vendor vendor
```

nit: my eyes struggled to find the directory from the sub-command, maybe make this clearly a relative directory usage like this?

```suggestion
    go-vendor ./vendor
```

---

That is certainly clearer. Since our documentation just has `go-vendor vendor`, I'm a bit torn.

---

🤷🏽 the usage can be different here and revised in the docs (if that's appropriate) to make it clearer that a directory is being passed as an argument. I'd vote to add the `./`, it is ultimately up to you here. It's only a nit :)